sockeye/training.py

```diff
@@ -455,7 +456,7 @@ def fit(self,
             mxmonitor_pattern: Optional[str] = None,
             mxmonitor_stat_func: Optional[str] = None,
             allow_missing_parameters: bool = False,
-            existing_parameters: Optional[str] = None):
+            existing_parameters: Optional[str] = None) -> 'TrainState':
```

Comment:
No need to forward reference as the TrainState is defined above, no?
```suggestion
            existing_parameters: Optional[str] = None) -> TrainState:
```

Reply:
thanks, indeed